src/VerificationCodeManagers/CacheVerificationCodeManager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\VerifyByPhone\VerificationCodeManagers;
+
+use Illuminate\Contracts\Cache\Repository;
+use Propaganistas\LaravelPhone\PhoneNumber;
+use Worksome\VerifyByPhone\Contracts\VerificationCodeGenerator;
+use Worksome\VerifyByPhone\Contracts\VerificationCodeManager;
+
+final class CacheVerificationCodeManager implements VerificationCodeManager
+{
+    public function __construct(
+        private Repository $cache,
+        private VerificationCodeGenerator $codeGenerator,
+    ) {
+    }
+
+    public function store(PhoneNumber $phoneNumber): string
+    {
+        $code = $this->codeGenerator->generate($phoneNumber);
```

Comment:
I feel like the verification code manager should not call the generator 🤔 

What if we call it a `VerificationCodeStore`, also to make it clearer that the purpose of it is to store a code and fetch it.